grammars/php.cson

```diff
@@ -178,12 +178,16 @@
   'comments':
     'patterns': [
       {
-        'begin': '/\\*\\*(?:#@\\+)?\\s*$'
-        'captures':
-          '0':
+        # This only highlights a docblock if the first line contains only two *s
+        # to stop highlighting everything as invalid when people do comment banners with /******** ...
+        'begin': '(/\\*\\*)(?=$|\\s+)'
```

Comment:
I guess not the \s should match a \n at the end.

We just don't want to match /***